easybuild/easyconfigs/r/RELION/RELION-3.0.7-foss-2019a.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '3.0.7'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['https://github.com/3dem/relion/archive']
+sources = [
+    '%(version)s.tar.gz',
+]
+patches = [
+    '%(name)s-3.0.4_no_which_cmd.patch',
+    '%(name)s-3.0.4_drop_usage_of_mpi_include_path.patch',
+    '%(name)s-3.0.5_add_hpc2n_submit_file_args.patch',  # This patch is a site specific patch. Either adapt or drop.
+    ('relion_sbatch.sh', '.'),  # This submit file template is used by the hpc2n_submit_file patch.
```

Comment:
@akesandgren `relion_sbatch.sh` is missing?

Also, isn't this site-specific?

Reply:
Yes it is somewhat site specific and can be left out of an upstream PR. But I do think it is useful for others, and it does not change anything unless the options are actually used.

Reply:
Rather doesn't change anything that matters. It gives you more fields to be filled out in the submit-job-screen

Reply:
We've installed 3.0.8 recently using this PR as the template. Having `relion_sbatch.sh` and `RELION-3.0.5_add_hpc2n_submit_file_args.patch` vastly simplified my job. I just had to update them to reflect our site specific differences and not workout what I needed to do from scratch.

Reply:
I'm a bit torn here... It's obviously useful, but since it requires some site-specific tweaking, I feel we can provide these files, but the use of them and the corresponding environment variables should be commented out mentioning some info on what may need to be tweaked?